File: go/3d-lotto-generator.go
package main

import (
  "fmt"
  "math/rand"
  "strings"
)

func generate3DigitNumber() string {
  digits := [3]int{rand.Intn(10), rand.Intn(10), rand.Intn(10)}
  return fmt.Sprintf("%d-%d-%d", digits[0], digits[1], digits[2])
}

func main() {
  var dataset []string
  
  // Generate 6 random three-digit numbers
  for i := 0; i < 6; i++ {
    dataset = append(dataset, generate3DigitNumber())
  }

  output := strings.Join(dataset, "\n")
  
  fmt.Println(output)
}

File: nodejs/3d-lotto-generator.js
function generate3DigitNumber() {
  const digits = [Math.floor(Math.random() * 10), Math.floor(Math.random() * 10), Math.floor(Math.random() * 10)];
  return digits.join("-");
}

const dataset = [];

// Generate 6 random three-digit numbers
for (let i = 0; i < 6; i++) {
  dataset.push(generate3DigitNumber());
}

let output = "";
for (const number of dataset) {
  output += `${number}\n`;
}

// Replace \n with <br>
output = output.replace(/\n/g, "<br>");

console.log(output);
